### PR TITLE
Update Java tools

### PR DESCRIPTION
**Pull request by lberki**

### CODE REVIEW

File: third_party/java/java_tools/README.md [hunk 1]
```diff
@@ -14,3 +14,35 @@ third_party/java/java_tools/turbine_direct_binary_deploy.jar
 third_party/java/java_tools/JavaBuilder_deploy.jar
 third_party/java/java_tools/jarjar_command_deploy.jar
 third_party/java/java_tools/Runner_deploy.jar
+
+The following tools were built with bazel 0.22.0 at commit 569e2cb4d1b7653b5a02bbf664765476233e6c33
```
Comment (philwo) [inline third_party/java/java_tools/README.md]: Why twice?

Comment (meisterT) [reply third_party/java/java_tools/README.md]: You may want to replace the file and not append it. You have the history in version control.

Comment (lberki) [reply third_party/java/java_tools/README.md]: /cc @iirina -- this was generated by your script.

Done.

**Comment by lberki** [comment]
Done!

**Comment by philwo** [comment]
Thanks! I'll import it.

**Comment by lberki** [comment]
Merci beaualot!

**Comment by philwo** [comment]
Pushed here: https://github.com/bazelbuild/bazel/commit/6aab2719d678057c4ef8edc3dfcad56803454732

**Pull request closed without merging**